extension/observer/ecsobserver/extension.go

```diff
@@ -15,15 +15,16 @@ var _ extension.Extension = (*ecsObserver)(nil)
 
 // ecsObserver implements component.ServiceExtension interface.
 type ecsObserver struct {
-	logger *zap.Logger
-	sd     *serviceDiscovery
+	logger            *zap.Logger
```

Comment:
Don't need logger, it is in `telemetrySettings.Logger`.

Reply:
Removed, thanks